brokerapi/brokers/account_managers/sql_account_manager.go

```diff
@@ -115,25 +127,27 @@ func (sam *SqlAccountManager) DeleteAccountFromGoogle(binding models.ServiceBind
 		return fmt.Errorf("Error creating CloudSQL client: %s", err)
 	}
 
-	op, err := sqlService.SslCerts.Delete(sam.ProjectId, instance.Name, sqlCreds.Sha1Fingerprint).Do()
-	if err != nil {
-		return fmt.Errorf("Error deleting ssl cert: %s", err)
-	}
+	if sqlCreds.CaCert != "" {
```

Comment:
I'd at least add a comment here about why. Or, this could be made clearer by retrieving the BindParameters and checking the generate_ssl_certs field.